### PR TITLE
added homework_14_03

### PR DESCRIPTION
**Pull request by Viktoriyadneprovskaya**

### CODE REVIEW

File: linkedincopy/mainpage.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    <form class="d-flex" role="search">
+        <a href="#">
+            <img sizes="" src="./src/pictures/LinkedIn_icon.svg" alt="logo"/>
+        </a>
+        <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+    </form>
+    <nav class="shift-menu">
+        <ul>
+            <li>
+                <div>
```
Comment (YeaaaaH) [inline linkedincopy/mainpage.html]: looks like you don't need additional div here(and the same li elements as well)

File: linkedincopy/src/test.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```
Comment (YeaaaaH) [inline linkedincopy/src/test.html:1]: remove it

**Review by YeaaaaH — APPROVED** [review]

File: linkedincopy/mainpage.html (hunk 1 above)
Comment (Viktoriyadneprovskaya) [reply linkedincopy/mainpage.html]: fixed

File: linkedincopy/src/test.html (hunk 2 above)
Comment (Viktoriyadneprovskaya) [reply linkedincopy/src/test.html:1]: done

**Pull request merged**